lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1621,7 +1621,7 @@ fc_extras
         attr_units = get_attr_units(cf_coord_var, attributes)
         points_data = cf_coord_var[:]
         # Gracefully fill points masked array.
-        if ma.isMaskedArray(points_data):
+        if (ma.isMaskedArray(points_data)) and (not (points_data.mask == False).all()):
```

Comment:
You know, I think you can achieve equivalent behaviour with `if ma.is_masked(points_data)`. Would you mind updating this line to use that masked array check instead?

Reply:
super nice pick up